### PR TITLE
SC-2640: Generation assets doesn't work if the destination folder wasn't created before generation

### PR DESCRIPTION
**Pull request by zyuzka**
Ticket: https://spryker.atlassian.net/browse/SC-2640

### CODE REVIEW

File: bin/export-helper.sh [hunk 1]
```diff
@@ -21,6 +33,7 @@ function doExport()
 
     case ${subCommand} in
         asset|assets)
+            isDestinationPathExist ${destinationPath}
```
Comment (sprymiker) [inline bin/export-helper.sh]: assertDestinationDirectory

Comment (zyuzka) [reply bin/export-helper.sh]: Fixed

File: bin/export-helper.sh [hunk 2]
```diff
@@ -1,4 +1,16 @@
-#!/usr/bin/env bash
+#!/bin/bash
+
+pushd "${BASH_SOURCE%/*}" > /dev/null
+. ./console.sh
+popd > /dev/null
+
+function isDestinationPathExist() {
+    if [ ! -d ${1} ];
```
Comment (sprymiker) [inline bin/export-helper.sh]:
```suggestion
    if [ ! -d ${1} ] || [ ! -w ${1} ];
```

File: bin/export-helper.sh [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+    then
+        error "${WARN}'${1}' path doesn\`t exist. Please, create target folder before run export command.${NC}"
```
Comment (sprymiker) [inline bin/export-helper.sh]:
```suggestion
        error "${WARN}Directory '${1}' is not accessible. Please, make sure it does exist and has appropriate permissions.${NC}"
```

**Comment by zyuzka** [comment]
> Please, fix the comments above.

Fixed

**Pull request merged**